x-pack/test/functional/page_objects/lens_page.ts

```diff
@@ -1377,21 +1377,28 @@ export function LensPageProvider({ getService, getPageObjects }: FtrProviderCont
      *
      * @param title - title for the new lens. If left undefined, the panel will be created by value
      * @param redirectToOrigin - whether to redirect back to the dashboard after saving the panel
+     * @param ignoreTimeFilter - whether time range has to be changed
      */
     async createAndAddLensFromDashboard({
       title,
       redirectToOrigin,
+      ignoreTimeFilter,
     }: {
       title?: string;
       redirectToOrigin?: boolean;
+      ignoreTimeFilter?: boolean;
     }) {
       log.debug(`createAndAddLens${title}`);
       const inViewMode = await PageObjects.dashboard.getIsInViewMode();
       if (inViewMode) {
         await PageObjects.dashboard.switchToEditMode();
       }
       await dashboardAddPanel.clickCreateNewLink();
-      await this.goToTimeRange();
+
+      if (ignoreTimeFilter === undefined || !ignoreTimeFilter) {
```

Comment:
In this PR `serverless_search` plugin creates default dataview with no timeFieldName, filtering all indices `*`. With this change dashboard won't be having the global time filter and hence FTR test fails. 
Adding this optional param to check if timeRange has to be changed.

Reply:
Nice! Nitpick: this can just be changed to ```if (!ignoreTimeFilter) {```